packages/babel-compiler/babel-compiler.js

```diff
@@ -1,5 +1,5 @@
 var semver = Npm.require("semver");
-
+var JSON5 = Npm.require("json5");
```

Comment:
For this to work in all environments, we need to add the `json5` npm dependency here:
https://github.com/meteor/meteor/blob/d1ca1f71c5992079836b36c4766394c7eab90e67/packages/babel-compiler/package.js#L12-L14
If you run `meteor test-packages ./babel-compiler` in the `meteor/packages` directory, that should automatically update the `meteor/packages/babel-compiler/.npm/package/npm-shrinkwrap.json` file, and those changes should be committed, too.

Reply:
@benjamn I applied the changes, but am somehow not able to run the test command - it always exits with:
```
=> Errors prevented startup:                  
   
   While selecting package versions:
   error: No version of babel-compiler satisfies all constraints: @=7.2.0-beta171.31, @~7.1.1
   Constraints on package "babel-compiler":
   * babel-compiler@=7.2.0-beta171.31 <- top level
   * babel-compiler@~7.1.1 <- top level
   * babel-compiler@7.1.0 <- ecmascript 0.11.1 <- allow-deny 1.1.0 <- mongo 1.5.0 <- autoupdate 1.4.1
   * babel-compiler@7.1.0 <- ecmascript 0.11.1 <- autoupdate 1.4.1
´´´